packages/craco-less/package.json

```
@@ -21,13 +21,13 @@
     "craco-less": "^2.0.0"
   },
   "devDependencies": {
-    "@craco/craco": "^6.4.3",
+    "@craco/craco": "^7.0.0-alpha.8",
     "@pika/pack": "^0.5.0",
     "@pika/plugin-build-node": "^0.8.1",
     "@pika/plugin-standard-pkg": "^0.8.1"
   },
   "peerDependencies": {
-    "@craco/craco": "^6.0.0",
+    "@craco/craco": "^7.0.0",
```

```suggestion
    "@craco/craco": "^7.0.0-alpha.8",
```

This should be `-alpha` currently as there is no stable currently.

---

v7 was released, not relevant anymore.